test/e2e/plain_provisioner_test.go

```diff
@@ -459,6 +459,55 @@ var _ = Describe("plain provisioner bundle", func() {
 			})
 		})
 	})
+
+	When("a bundle containing nested directry is created", func() {
+		var (
+			bundle *rukpakv1alpha1.Bundle
+			ctx    context.Context
+		)
+		BeforeEach(func() {
+			ctx = context.Background()
+
+			By("creating the testing Bundle resource")
+			bundle = &rukpakv1alpha1.Bundle{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "olm-crds-unsupported",
```

Comment:
super nit: can we change the name as there's nothing related to OLM in the testdata manifests?

Reply:
Yes, I'll update.  Thanks!